contrib/.rubocop.yml

```diff
@@ -0,0 +1,7 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
why do we have a `.rubocop.yml` file in `contrib/` and another one in `dist/`? You can just have one, you only need to split the rails directory... :confused: